packages/core/src/routes/session/social.test.ts

```diff
@@ -1,10 +1,12 @@
+/* eslint-disable max-lines */
```

Comment:
can we split this file?

Reply:
Removed duplicated UT of `POST /session/register/social` (it has already been moved to packages/core/src/routes/session/social.bind-social.test.ts but forgot to be removed).